payments/models.py

```diff
@@ -813,7 +821,7 @@ def __str__(self):
 class OrderLogEntry(UUIDModel, TimeStampedModel):
     order = models.ForeignKey(
         Order,
-        verbose_name=_("order log entry"),
```

Comment:
Not sure why this is changed.

Reply:
I realised it was the wrong value, since it refers to `order`, not the log itself.

Reply:
Oh right, missed that as well.